spark/src/main/java/org/apache/iceberg/spark/source/Writer.java

```diff
@@ -24,6 +24,35 @@
 import com.google.common.collect.Iterables;
 import com.google.common.collect.Lists;
 import com.google.common.collect.Sets;
+import org.apache.iceberg.AppendFiles;
```

Comment:
Why were all these imports added? Are they used? The rest of the changes are small.

Reply:
I don't think these are used. I've removed them. Thanks for pointing this out.